### PR TITLE
Alex Yastrebov 

### PR DESCRIPTION
**Pull request by OleksandrYastrebov**
created file

### CODE REVIEW

File: styles/Alex-Yastrebov.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
+body,html{
+height:100%;
```
Comment (kate-sagittarius) [inline styles/Alex-Yastrebov.css]: indents

File: styles/Alex-Yastrebov.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+/* background-color: #7A41E8; */
+background-color: #27265E;
+color:#fff;
+
+}
+/* .wrapper{
+  margin-top: 0px;
```
Comment (kate-sagittarius) [inline styles/Alex-Yastrebov.css]: try avoiding pushing commenting code, it is not a very nice practice

File: team-members/Alex-Yastrebov.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+   <head>
+     <meta charset="utf-8">
+     <title></title>
```
Comment (kate-sagittarius) [inline team-members/Alex-Yastrebov.html]: please, add title

File: team-members/Alex-Yastrebov.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+      <main>
+
+        <section class='about_wrapper'>
+          <div class="about">
+            <h1>About</h1>
+            </div>
+            <br>
+           <p id="about_text">
+  The size of our homes shapes so much of our lives.
```
Comment (kate-sagittarius) [inline team-members/Alex-Yastrebov.html]: indents

**Review by kate-sagittarius — CHANGES_REQUESTED** [review]
Please, check and correct the file and then we will merge

File: styles/Alex-Yastrebov.css (hunk 1 above)
Comment (OleksandrYastrebov) [reply styles/Alex-Yastrebov.css]: done? )

File: styles/Alex-Yastrebov.css (hunk 2 above)
Comment (OleksandrYastrebov) [reply styles/Alex-Yastrebov.css]: done

File: team-members/Alex-Yastrebov.html (hunk 3 above)
Comment (OleksandrYastrebov) [reply team-members/Alex-Yastrebov.html]: done

File: styles/Alex-Yastrebov.css (hunk 1 above)
Comment (OleksandrYastrebov) [reply styles/Alex-Yastrebov.css]: Какие отступы?

File: team-members/Alex-Yastrebov.html (hunk 4 above)
Comment (OleksandrYastrebov) [reply team-members/Alex-Yastrebov.html]: done

File: styles/Alex-Yastrebov.css (hunk 1 above)
Comment (kate-sagittarius) [reply styles/Alex-Yastrebov.css]: css should look like this:
```
body, html {
  height: 100%;
}
````

Comment (OleksandrYastrebov) [reply styles/Alex-Yastrebov.css]: done

**Pull request merged**